forest/cli/src/cli/send_cmd.rs

```diff
@@ -7,9 +7,69 @@ use forest_json::message::json::MessageJson;
 use forest_rpc_client::{mpool_push_message, wallet_default_address};
 use fvm_shared::{address::Address, econ::TokenAmount, message::Message, METHOD_SEND};
 use num::BigInt;
+use rust_decimal::prelude::*;
+use rust_decimal_macros::dec;
 
 use super::{handle_rpc_err, Config};
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct FILAmount {
+    pub value: TokenAmount,
+}
+
+impl FromStr for FILAmount {
+    type Err = anyhow::Error;
+
+    /// `FILAmount::from_str` parses a string (input to the 'amount' field of
+    /// the the `send` command) with all units supported by forest wallet
+    /// (`attoFIL`, `femtoFIL`, `picoFIL`, `nanoFIL`, `microFIL`,
+    /// `milliFIL`, and `FIL`) and converts the amount into an `attoFIL`
+    /// `TokenAmount`. To match the current behavior in Lotus, the default
+    /// units (if no units are specified in the command line) are `FIL`.
```

Comment:
The documentation should describe what this function does. This means you should not mention the `send` command at all. That this function happens to be used by the `send` command is irrelevant.

Describe what this function does. Show some positive examples. Show some negative examples. Talk about constraints, the relationship to `format_balance_string`, and mention that we want to accept all input accepted by Lotus as a footnote.